### PR TITLE
adding keypoints to 02

### PR DESCRIPTION
**Pull request by wrightaprilm**
Add keypoints for lesson release

### CODE REVIEW

File: _episodes/02-starting-with-data.md [hunk 1]
```diff
@@ -18,7 +18,10 @@ objectives:
     - "Perform basic mathematical operations and summary statistics on data in a Pandas DataFrame."
     - "Create simple plots."
 keypoints:
-    - "FIXME"
+    - "Libraries enable you to add functions to Python. Pandas is a popular library for working with data."
```
Comment (maxim-belkin) [inline _episodes/02-starting-with-data.md]: I suggest splitting this into two:
1. Python libraries extend the functionality of Python.
2.

Comment (maxim-belkin) [reply _episodes/02-starting-with-data.md]: ...Classic mistake (hitting Return before completing the message)...
2. Pandas is a popular Python library for ....

Comment (wrightaprilm) [reply _episodes/02-starting-with-data.md]:
```suggestion
    - "Libraries enable you to add functions to Python." 
    - "Pandas is a popular library for working with data."
```

File: _episodes/02-starting-with-data.md [hunk 2]
```diff
@@ -18,7 +18,11 @@ objectives:
     - "Perform basic mathematical operations and summary statistics on data in a Pandas DataFrame."
     - "Create simple plots."
 keypoints:
-    - "FIXME"
+    - "Libraries enable you to add functions to Python." 
+    - "Pandas is a popular library for working with data."
+    - "A DataFrame can be used to access data either by column name, column integer, or row."
```
Comment (maxim-belkin) [inline _episodes/02-starting-with-data.md]: Dataframe is a Pandas data structure that allows one to access data by column (name or index) or row.

File: _episodes/02-starting-with-data.md [hunk 3]
```diff
@@ -18,7 +18,11 @@ objectives:
     - "Perform basic mathematical operations and summary statistics on data in a Pandas DataFrame."
     - "Create simple plots."
 keypoints:
-    - "FIXME"
+    - "Libraries enable you to add functions to Python."
```
Comment (maxim-belkin) [inline _episodes/02-starting-with-data.md]: I'm not sure we should say that libraries enable us to add functions... They allow us extend the limits of what Python can do...

File: _episodes/02-starting-with-data.md (hunk 2 above)
Comment (wrightaprilm) [reply _episodes/02-starting-with-data.md]:
```suggestion
    - "A Dataframe is a Pandas data structure that allows one to access data by column (name or index) or row."
```

File: _episodes/02-starting-with-data.md (hunk 3 above)
Comment (wrightaprilm) [reply _episodes/02-starting-with-data.md]: Oh, that's a good point - you could add your own functions without installing any libraries.

Comment (wrightaprilm) [reply _episodes/02-starting-with-data.md]:
```suggestion
    - "Libraries enable us to extend the functionality of Python." 
```

Comment (wrightaprilm) [reply _episodes/02-starting-with-data.md]: Maybe that?

Comment (maxim-belkin) [reply _episodes/02-starting-with-data.md]: 👍

File: _episodes/02-starting-with-data.md [hunk 4]
```diff
@@ -18,7 +18,11 @@ objectives:
     - "Perform basic mathematical operations and summary statistics on data in a Pandas DataFrame."
     - "Create simple plots."
 keypoints:
-    - "FIXME"
+    - "Libraries enable us to extend the functionality of Python." 
+    - "Pandas is a popular library for working with data."
+    - "A Dataframe is a Pandas data structure that allows one to access data by column (name or index) or row."
+    - "Aggregating data using the `groupby()` function enables you to generate useful summaries of data quickly."
+    - "Plots can be created from DataFrames or from subsets of data that have been generated with `groupby()`."
```
Comment (maxim-belkin) [inline _episodes/02-starting-with-data.md]: `from DataFrames or from subsets of data` -> `from DataFrames or subsets of data`

Comment (wrightaprilm) [reply _episodes/02-starting-with-data.md]:
```suggestion
    - "Plots can be created from DataFrames or subsets of data that have been generated with `groupby()`."
```

**Comment by wrightaprilm** [comment]
OK, if you're approved, I'll merge!

**Pull request merged**